src/main/java/com/internet/shop/service/impl/ProductServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package com.internet.shop.service.impl;
+
+import com.internet.shop.dao.ProductDao;
+import com.internet.shop.lib.Inject;
+import com.internet.shop.lib.Service;
+import com.internet.shop.model.Product;
+import com.internet.shop.service.ProductService;
+import java.util.List;
+
+@Service
+public class ProductServiceImpl implements ProductService {
+    @Inject
+    private ProductDao productDao;
+
+    @Override
+    public Product create(Product product) {
+        return productDao.create(product);
+    }
+
+    @Override
+    public Product getById(Long productId) {
+        return productDao.getById(productId).orElseThrow();
```

Comment:
```suggestion
        return productDao.getById(productId).get();
```
let's use `get()` by now

Reply:
done